### PR TITLE
List Zero (ZER)

### PR DESCRIPTION
**Pull request by tearodactyl**

### CODE REVIEW

File: src/main/java/bisq/asset/coins/Zero.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        super("Zero", "ZER", new ZeroAddressValidator());
+    }
+
+
+    public static class ZeroAddressValidator implements AddressValidator {
+
+        @Override
+        public AddressValidationResult validate(String address) {
+            // We only support t addresses (transparent transactions)
+            if (!address.startsWith("t1"))
```
Comment (blabno) [inline src/main/java/bisq/asset/coins/Zero.java]: Is that the only validation requirement?

Is "t" address valid?

Comment (R3DEY3UK) [reply src/main/java/bisq/asset/coins/Zero.java]: yes that is the only "Validation requirement".

Comment (tearodactyl) [reply src/main/java/bisq/asset/coins/Zero.java]: For us, same as Zcash, only transactions to T addresses can be readily verified. and valid addresses start with "t1".

**Comment by cbeams** [comment]
Closing as incorrect.

**Pull request closed without merging**